### PR TITLE
AARA-393 Phone Number Fix in RFA 01A Section IX: References

### PR DESCRIPTION
**Pull request by nasar-cambria**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/AARA-393

#### Brief feature description
- Display of Phone Number in RFA 01A Section IX: References is inconsistent. Display should match Applicant phone number (999) 999-9999

#### Does this PR block/effect other stories?
- No

#### Do any commands needs to be run in order to run the project with this PR?
- No

#### Code coverage percent diff as a result of this PR?
- NA

### CODE REVIEW

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Comment by lauherkcambria** [comment]
jenkins re-test this please

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Comment by lauherkcambria** [comment]
jenkins re-test this please

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Comment by nasar-cambria** [comment]
jenkins re-test this please

**Comment by anuroop15** [comment]
jenkins retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

File: app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default class ReferencesCard extends React.Component {
             blurPlaceholder=''
             focusPlaceholder='(___)___-____'
             options={{
-              delimiters: ['(', ')', '-'],
+              delimiters: ['(', ') ', '-'],
```
Comment (ratneshraval) [inline app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx]: Look at other delimiters we've used, you need space as separate one. 
This used to create an issue with when trying delete digits with backspace. Not sure if it is still the case.

Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/referencesCard.jsx]: Good catch @ratneshraval - Done.

**Pull request merged**